assets/collapsible-content.css

```diff
@@ -0,0 +1,104 @@
+.layout-section-container {
+  padding-bottom: 5rem;
+  padding-top: 5rem;
+}
+
+@media screen and (min-width: 750px) {
+  .layout-section-container {
+    padding-bottom: 7rem;
+    padding-top: 7rem;
+  }
+}
+
+.collapsible-content__media--small {
+  height: 19.4rem;
+}
+
+.collapsible-content__media--large {
+  height: 43.5rem;
+}
+
+@media screen and (min-width: 750px) {
+  .collapsible-content__media--small {
+    height: 31.4rem;
+  }
+
+  .collapsible-content__media--large {
+    height: 69.5rem;
+  }
+}
+
+.collapsible-content__grid, .collapsible-content__grid .grid  {
+  margin-left: 0;
+}
+
+.collapsible-content-wrapper {
+  padding-top: 5rem;
+  padding-bottom: 3rem;
+}
+
+@media screen and (min-width: 750px) {
+  .collapsible-content__grid--reverse {
+    flex-direction: row-reverse;
+  }
+
+  .collapsible-content-wrapper {
+    padding-top: 7rem;
+    padding-bottom: 4rem;
+  }  
+}
+
+.collapsible-content-wrapper--narrow {
+  margin: 0 auto;
+  padding-right: 1.5rem;
+  padding-left: 1.5rem;
+  max-width: 73.4rem;
+}
+
+.layout-row-container .accordion:not(:first-child):not(.color-background-1) {
+  margin-top: 1rem;
+}
+
+.caption-with-letter-spacing + h2 {
+  margin-top: 1rem;
+}
+
+@media screen and (min-width: 750px) {
+  .collapsible-content .accordion {
+    margin-top: 0;
+  }
+}
+
+.layout-row-container .accordion {
```

Comment:
I find the class naming a little too generic and not specific enough to the context. 

`layout-row-container` vs `collapsible-row-layout` for example.